src/ioctls/vcpu.rs

```diff
@@ -2172,4 +2205,35 @@ mod tests {
             vcpu.enable_cap(&cap).unwrap();
         }
     }
+    #[cfg(target_arch = "x86_64")]
+    #[test]
+    fn test_get_tsc_khz() {
```

Comment:
What is the reason for having this as a separate test? I was thinking qe can test both the setter and getter in the same test (something like `test_tsc_khz`) because otherwise this test seems trivial, and the functionality would be covered in  `test_set_tsc_khz`.

Reply:
I think it's fine here to have separate tests since on some machines (like my laptop for example) `GetTscKhz` is present and `TscControl` is missing.
We should anyway test in `test_set_tsc_khz` that `get_tsc_khz` returns the value that was just set.

Reply:
Done.